packages/pds/src/db/tables/record.ts

```diff
@@ -4,7 +4,6 @@ export interface Record {
   did: string
   collection: string
   rkey: string
-  raw: string
   receivedAt: string
   indexedAt: string
```

Comment:
we have `indexedAt` in a lot of places:
- blockstore
- this record table
- & the structure table

wondering if we should simplify this or leave as is 🤔

Reply:
I am totally pickin' up what you're putting down.  Just one idea— maybe we keep it on the blockstore and on the secondary indexes, but remove it from the record table.  My thought there is that the secondary index tables technically could end-up combining data from multiple records, and we'd want to keep the indexed time of the latest data.  The record table, on the other hand, is pretty much guaranteed to reference a single block.

Reply:
seems reasonable to me 👌